### DISCUSSION TITLE
How to disable triggering HEAD request method when fetching the translation files with next-i18next

### DISCUSSION THREAD

**Discussion by Selnia** [i18next/next-i18next#2106] [Q&A]
The next-i18next is using HEAD request method when matching the previous translation file if it's still the same or if it still matches

My code looks like this:
```
import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
import { GetStaticPropsContext } from 'next'

export async function getStaticProps({ locale = 'en' }: GetStaticPropsContext) {
  return {
    props: {
      ...(await serverSideTranslations(locale, ['common'])),
    },
  }
}
```

How can I disable the HEAD request method and use GET request instead when next-i18next is matching the translation files?

because apparently, our server has the HEAD request method disabled so I cannot use the HEAD request method in our project

I want to make the HEAD request also become a GET request

**Comment by adrai** [comment]
You may ask over at https://github.com/vercel/next.js
